RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,570 @@
+        "429":
+          description: The client issued too many requests in a short period of time.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+  /ris/v1/resource_details/{entityid}:
+    summary: Query detailed information of a resource from RIS. 
+    get:
+      tags:
+      - Resource references
+      - Resource capacities
+      - Resource definition
+      - Resource status
+      summary: Retrieve the specified operational intent details from a PSU.
+      description: |-
+        The RIS hosting this endpoint returns the details (and reference) of an
+        resource it manages.
+      operationId: getResourceDetailsChanged
+      parameters:
+      - name: entityid
+        in: path
+        description: EntityID for this resource.
+        required: true
+        style: simple
+        explode: false
+        schema:
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/EntityID'
+      responses:
+        "200":
+          description: Operational intent details were retrieved successfully.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/GetResourceDetailsResponse'
+        "400":
+          description: One or more input parameters were missing or invalid.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "401":
+          description: "Bearer access token was not provided in Authorization header, token could not be decoded, or token was invalid."
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "403":
+          description: The access token was decoded successfully but did not include a scope appropriate to this endpoint.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "404":
+          description: The requested Entity could not be found.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+        "429":
+          description: The client issued too many requests in a short period of time.
+          content:
+            application/json:
+              schema:
+                $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/ErrorResponse'
+      security:
+      - Authority:
+        - utm.strategic_coordination
+
+  # Renamed capacity to future_capacities? Thoughts?
```

Comment:
Doesn't future_capacities imply the start time couldn't be now, or even in the past? I think "capacities" may be more general.

Reply:
I was trying to make it more clear that GET capacities doesn't return the default capacity (that is part of resource details) and doesn't return the current capacity (part of status).  It only returns the future capacities. Since the capacity details are spread across different endpoints, I'm worried "capacities" may not be very clear.  I named many of the schemas with "Future" to try and clarify. I'm open to whatever here, just trying to point out the concern.

Reply:
Good point. Future may be the way to go. Or we need to think of a way to simplify so its not confusing for the user.